src/SignalR/clients/java/signalr/src/main/java/com/microsoft/signalr/NegotiateResponse.java

```diff
@@ -28,6 +28,10 @@ public NegotiateResponse(String negotiatePayload) {
                     case "error":
                         this.error = reader.nextString();
                         break;
+                    case "ProtocolVersion":
+                        this.error = "Detected a connection attempt to an ASP.NET SignalR Server. This client only supports connecting to an ASP.NET Core SignalR Server. See https://aka.ms/signalr-core-differences for details.";
```

Comment:
"Detected a connection attempt" sounds a little awkward and I think we can simplify a bit.

```suggestion
                        this.error = "Detected an ASP.NET SignalR Server. This client only supports connecting to an ASP.NET Core SignalR Server. See https://aka.ms/signalr-core-differences for details.";
```